zkfacade/src/main/java/com/yahoo/vespa/curator/SingletonManager.java

```diff
@@ -302,9 +306,20 @@ private void renewLease() {
             }
         }
 
+        private void cleanOrphans() {
```

Comment:
```suggestion
        /** This is a work-around for a bug where Curator leaks lock nodes, through "protection mode", probably because unexpected KeeperException children are thrown? */
        private void cleanOrphans() {
```